src/ice.c

```diff
@@ -3989,28 +3990,31 @@ static gboolean janus_ice_outgoing_transport_wide_cc_feedback(gpointer user_data
 		while (g_hash_table_iter_next(&iter, NULL, &value)) {
 			janus_ice_peerconnection_medium *m = value;
 			if(m && m->type == JANUS_MEDIA_VIDEO && m->recv) {
-				medium = m;
-				break;
+				/* If a medium (or simulcast layer, if applicable) has not received data, its SSRC may be unknown. */
+				/* Pick the first valid SSRC we find across all considered mediums */
+				int i = 0;
+				for(i = 0; i < 3; i++) {
+					if(m->ssrc_peer[i] != 0)
+						break;
+				}
+
+				/* Stop if we found a valid SSRC/medium to use */
+				if(i < 3) {
+					ssrc_peer = m->ssrc_peer[i];
+					medium = m;
+					break;
+				}
 			}
 		}
 		janus_mutex_unlock(&handle->mutex);
 	}
 
-	if(pc && pc->do_transport_wide_cc && medium) {
-		/* Simulcast layers that have not received data will not have a recorded peer SSRC. */
-		/* Pick the first layer that has a peer SSRC */
-		guint32 ssrc_peer = 0;
-		for(int i = 0; i < 3; i++) {
-			if(medium->ssrc_peer[i] != 0) {
-				ssrc_peer = medium->ssrc_peer[i];
-				break;
-			}
-		}
-		if (ssrc_peer == 0) {
-			JANUS_LOG(LOG_WARN, "No peer SSRC, cannot send transport-wide CC feedback\n");
-			return G_SOURCE_CONTINUE;
-		}
+	if(!medium) {
```

Comment:
Just as a small nit and personal preference in terms of code style, we typically use `if(medium == NULL)` for checks like that.

Reply:
I found instances of both styles in the code base and wasn't sure which to use. Is there a coding style document I could refer to in the future?

Reply:
You're 100% right, and apologies for the confusion: that's mainly because of the many contributions we've received so far (and of which I'm grateful for of course), where we haven't always enforced a specific code style. Unfortunately we don't have a reference for that, and I'm not really sure we'll prepare one: even @atoppi and I write things a bit differently :laughing:  These are small nits that can be ascribed to my excessive pedantry on the matter, so no need to concern too much on that!